backend/tests/api/routes/test_ideas.py

```diff
@@ -916,3 +937,55 @@ async def test_POST_create_idea_returns_idea(
         assert data["upvoted_by"] == []
         assert data["downvoted_by"] == []
         assert data["creator_id"] == str(user.id)
+
+
+@pytest.mark.integration
+@pytest.mark.anyio
+async def test_POST_create_idea_creates_idea_in_db(
+    real_db: AIOSession,
+    user_with_client,
+):
+    fake = faker.Faker()
+    user, async_client = user_with_client
+
+    test_idea_create = {
+        "name": fake.sentence(nb_words=5, variable_nb_words=True),
+        "description": fake.paragraph(nb_sentences=5, variable_nb_sentences=True),
+    }
+
+    async with clean_new_ideas(real_db):
+        response = await async_client.post(
+            "/ideas/",
+            json=test_idea_create,
+        )
+
+        data = response.json()
+
+        assert response.status_code == 200
+
+        db_idea = await real_db.find_one(Idea, Idea.id == ObjectId(data["id"]))
+
```

Comment:
```suggestion

assert db_idea is not None
```
A bit nitpicky, to make type checker stop freaking out, because `db_idea` can be `None`, if no entry was found. And asserts below _might_ be checking specified attributes on `None`.